### PR TITLE
Support original version strings for HammerDB

### PR DESCRIPTION
**Pull request by portante**
We are a bit forgiving of the "Hammerdb-tpcc" version string, accepting an empty string, and a lowercase "db" in "HammerDB-3.2".

### CODE REVIEW

**Review by portante — COMMENTED** [review]
Ready for review.

**Review by dbutenhof — APPROVED** [review]
Looks plausible, although a lot rests on the changes in `test-7.24.tar.xz`, which I haven't investigated. However, the new test output certainly suggests that you've hit the changed code.

**Review by dbutenhof — APPROVED** [review]

**Review by ndokos — DISMISSED** [review]

File: lib/pbench/server/indexer.py [hunk 1]
```diff
@@ -847,15 +847,23 @@ def _known_user_benchmark(ptb):
...
             pass
         else:
-            ver = ver.strip()
+            # For compatibiilty with "Hammerdb-tpcc", we'll map an empty
+            # string and "Hammerdb-3.2" to the expected value of
+            # "HammerDB-3.2".
+            if name == "Hammerdb-tpcc" and (not ver or ver == "Hammerdb-3.2"):
+                ver = "HammerDB-3.2"
+            else:
+                ver = ver.strip()
```
Comment (dbutenhof) [inline lib/pbench/server/indexer.py:866]: Why are you re-stripping `ver` here? You already did it at 854 unless you set it to `""` at 851??

Comment (portante) [reply lib/pbench/server/indexer.py]: Fixed.

**Review by dbutenhof — APPROVED** [review]

**Review by dbutenhof — APPROVED** [review]

**Review by npalaska — APPROVED** [review]

**Pull request merged**